torchao/prototype/moe_training/scaled_grouped_mm.py

```diff
@@ -294,6 +294,7 @@ def forward(
         out_dtype: Optional[torch.dtype] = torch.bfloat16,
         emulated: bool = False,
         use_triton_for_dim0_cast: bool = False,
+        wgrad_with_hp: bool = False,
```

Comment:
we already use naming `with_gw_hp`, do you want to match it?

https://github.com/pytorch/ao/blob/1f9bfd7ebf054c616543052c5767a8e6972cbdf8/torchao/float8/config.py#L135